.github/workflows/update-requirements.yml

```
@@ -94,10 +95,24 @@ jobs:
           name: requirements.txt
           path: requirements
 
+      - name: Commit requirements files
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add .
+          git commit -m "Update requirements files." -a || echo "Nothing to update"
+
+      - name: Push changes
+        uses: ad-m/github-push-action@v0.5.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: ${{ github.ref }}
+
       - name: Create Pull Request
```

```suggestion
      - name: Create Pull Request (since update via push failed)
```
(if I understand correctly the purpose of this step)